tests/Client/TestRepo.php

```diff
@@ -34,43 +29,41 @@ class TestRepo implements RepoFileFetcherInterface
      */
     public function __construct(string $fixturesSet)
     {
-        $this->fixturesSet = $fixturesSet;
+        // Store all the repo files locally so they can be easily altered.
+        // @see self::setRepoFileNestedValue()
+        $repoFiles = glob(static::getFixturesRealPath($fixturesSet, '/tufrepo/metadata') . '/*.json');
+        foreach ($repoFiles as $repoFile) {
+            $this->repoFilesContents[basename($repoFile)] = file_get_contents($repoFile);
+        }
     }
 
     /**
      * {@inheritdoc}
      */
     public function fetchFile(string $fileName, int $maxBytes)
     {
-        try {
-            $contents = file_get_contents(__DIR__ .  "/../../fixtures/{$this->fixturesSet}/tufrepo/metadata/$fileName");
-            if ($contents === false) {
-                throw new RepoFileNotFound("File $fileName not found.");
-            }
-            // Alter the signed portion of the json contents to trigger an
-            // exception.
-            // @see \Tuf\Client\Updater::checkSignatures()
-            if (in_array($fileName, $this->failSignatureFiles)) {
-                $json = json_decode($contents, true);
-                $json['signed']['extra_test_value'] = 'value';
-                $contents = JsonNormalizer::asNormalizedJson($json);
-            }
-            return $contents;
-        } catch (\Exception $exception) {
-            return false;
```

Comment:
It was suppose to throw an exception if the file is not found but it was returning false. Obviously we are at the point where we could change this if we wanted to but I think throwing the exception is good.